packages/amplify/amplify_flutter/pubspec.yaml

```diff
@@ -8,6 +8,7 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
+  amplify_logger: ^0.1.0
```

Comment:
I don't think this needs its own package right now. I would move this to `amplify_core`.